code/hud/hud.cpp

```diff
@@ -2021,7 +2021,7 @@ void hud_damage_popup_init()
 	Damage_flash_timer =	1;
 
 	for ( i = 0; i < SUBSYSTEM_MAX; i++ ) {
-		Pl_hud_subsys_info[i].last_str = 1000.0f;
```

Comment:
Do you know why it was initialized to 1000? Seems like a strange value to initialize 'last strength' to...